src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_help.py

```diff
@@ -378,3 +383,31 @@
         - name: Delete Geo-Disaster Recovery Configuration Alias
           text: az eventhubs georecovery-alias delete --resource-group myresourcegroup --namespace-name secondarynamespace --alias myaliasname
 """
+
+helps['eventhubs namespace network-rule add'] = """
+    type: command
+    short-summary: Add a network rule for a namespace.
+    examples:
+        - name: add a VirtualNetwork rule in NetworkruleSet for a namespace
+          text: az eventhubs namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace --subnet subnetId --ignore-missing-endpoint True
+        - name: add a IP rule in NetworkruleSet for a namespace
+          text: az eventhubs namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace --ip-address ipmask --action Allow
```

Comment:
`--ip-address` should be an actual value.

Reply:
resolved